lib/Backend/SccLiveness.cpp

```diff
@@ -722,7 +722,7 @@ Lifetime *
 SCCLiveness::InsertLifetime(StackSym *stackSym, RegNum reg, IR::Instr *const currentInstr)
 {
     const uint start = currentInstr->GetNumber(), end = start;
-    Lifetime * newLlifetime = JitAnew(tempAlloc, Lifetime, tempAlloc, stackSym, reg, start, end, this->func);
```

Comment:
Could you explain this change? The rest looks like standard "make sure to initialize everything and do it in declaration order" stuff, but this seems different.

Reply:
Oh, yeah. When doing the pass over the initializers I noticed that the Func* argument is never used, so I removed it.

Reply:
I had to wander through macros for a while to find it, but the varargs here are passed to the second parameter of JitAnew (Lifetime in this case). Lifetime had its last constructor parameter removed because it was unused.